quickstart/dotnet-core/README.md

```diff
@@ -11,33 +11,55 @@ products:
 ---
 # Quickstart: Use Azure Cache for Redis in .NET Core
 
-This sample shows you how to incorporate Azure Cache for Redis into a C# .NET Core console app using the [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis) Redis client. See the [accompanying article](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-dotnet-core-quickstart) on the documentation site for details, including best practices and how to create the sample code from scratch.
+See the [accompanying article](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-dotnet-core-quickstart) on the documentation site for details, including best practices and how to create the sample code from scratch.
 
 ## Prerequisites
 
 - Azure subscription - [create one for free](https://azure.microsoft.com/free/)
 - Azure Cache for Redis cache - [create one](https://docs.microsoft.com/azure/azure-cache-for-redis/quickstart-create-redis)
 - [.NET Core SDK](https://dotnet.microsoft.com/download)
+- [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/)
```

Comment:
Azure Developer CLI isn't really required, right? The az CLI should be sufficient

Reply:
If we add bicep files and azd support, using Azure Developer CLI will be more consistent.